pkg/util/config/config.go

```diff
@@ -66,12 +66,20 @@ type Scalyr struct {
 	ScalyrMemoryLimit   string `name:"scalyr_memory_limit" default:"1Gi"`
 }
 
+// LogicalBackup
+type LogicalBackup struct {
+	LogicalBackupSchedule    string `name:"logical_backup_schedule" default:"30 00 * * *"`
+	LogicalBackupDockerImage string `name:"logical_backup_docker_image" default:"registry.opensource.zalan.do/acid/logical-backup"`
```

Comment:
is this valid without a tag?

Reply:
yes, this will fetch the image tagged `:latest`. This tag does [have its own problems](https://vsupalov.com/docker-latest-tag/), however in that particular case it should always fetch the last build image. That is exactly the image we intend to use ourselves.